### PR TITLE
Can has data?

### PR DESCRIPTION
**Pull request by HawesDomingue**

### CODE REVIEW

**Comment by marshallgallatin** [comment]
I don't know whats really going on here, I defer to Josh

File: app/models.py [hunk 1]
```diff
@@ -191,19 +196,20 @@ class NutritionalContent(Base):
...
-    sodium_in_milligrams         = Column(Integer, isNonnegative('sodium_in_milligrams')        , nullable=False)
-    total_carbohydrates_in_grams = Column(Integer, isNonnegative('total_carbohydrates_in_grams'), nullable=False)
-    dietary_fiber_in_grams       = Column(Float,   isNonnegative('dietary_fiber_in_grams')      , nullable=False)
-    sugar_in_grams               = Column(Float,   isNonnegative('sugar_in_grams')              , nullable=False)
-    protein_in_grams             = Column(Float,   isNonnegative('protein_in_grams')            , nullable=False)
-    vitamin_a_in_iu              = Column(Integer, isNonnegative('vitamin_a_in_iu')             , nullable=False)
-    vitamin_c_in_milligrams      = Column(Integer, isNonnegative('vitamin_c_in_milligrams')     , nullable=False)
-    calcium_in_milligrams        = Column(Integer, isNonnegative('calcium_in_milligrams')       , nullable=False)
-    iron_in_milligrams           = Column(Integer, isNonnegative('iron_in_milligrams')          , nullable=False)
+    calories = Column(Integer, isNonnegative('calories'), nullable=False)
```
Comment (Joshie196) [inline app/models.py]: I would format this like before so its more readable

**Comment by Joshie196** [comment]
Other than my formatting comment, looks good. Ship it!

**Comment by HawesDomingue** [comment]
Restructured database slightly, added first round of real data.

**Pull request merged**